### DISCUSSION TITLE
How to stop event propagation?

### DISCUSSION THREAD

**Discussion by ASamSam** [geoman-io/leaflet-geoman#1544] [Questions & Help]
Hello. Can you please help me figure out how to stop event propagation for the particular point on the polyline (endpoints)?
I do use polyline.on('pm:markerdragstart', (e).
found, that if the exception happens in the event - it stops, looking for a more accurate way

**Comment by Falke-Design** [comment]
Saddly this is the behavior of Leaflet events not from Geoman. The only workaround is to implement to each listener a try / catch block
